programmers/난이도별/level01.같은_숫자는_싫어/sjjyy.java

```diff
@@ -0,0 +1,33 @@
+package 난이도별.level01.같은_숫자는_싫어;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class sjjyy
+{
+    public static int [] solution(int[] array)
+    {
+        ArrayList<Integer> arrayList = new ArrayList<>();
```

Comment:
```suggestion
        List<Integer> arrayList = new ArrayList<>();
```
```suggestion
        List<Integer> arrayList = new ArrayList<>();
```
List로 해줘도 상관 없어요~
List는 interface, ArrayList는 구현체(implement)라서 List의 확장성이 더 뛰어납니다!